app/views/selectRoom.html

```diff
@@ -1,4 +1,4 @@
-<div>
+<div class="col-md-offset-3 col-md-6">
```

Comment:
Had to reset the styling to equal the mood overview, which was wrongly styled because it was loaded in a nested view instead of the correct view.